### PR TITLE
fix: images being bundled through webpack

### PR DESCRIPTION
**Pull request by mhomolak**
Images should now be visible for teams consuming app-header in an application.

### CODE REVIEW

**Comment by mhomolak** [comment]
@wyseguyonline @StommePoes @umahaea

File: src/html/header.template [hunk 1]
```diff
@@ -5,13 +5,13 @@
 				<if condition="data.mode === 'Basic' || data.mode === 'Legacy Course'">
 					<a href="{data.links.home}">
 						<div class="o-header__logo o-header__logo--pearson">
-							<img src="../../img/logo_large.png" alt="console home">
+							<img src="../../build/img/logo_large.png" alt="console home">
```
Comment (wyseguyonline) [inline src/html/header.template]: Just want to be sure that we've discussed this with Console (Hung) and they're ok with having a build/img directory?

**Comment by wyseguyonline** [comment]
We should also add to the documentation that the .png files will need to be stored in a specific location on the consuming app's server.

**Comment by mhomolak** [comment]
After talking with Hung, I am going to see exactly what he needs to do (because we're both not 100% sure) and will add any necessary instructions to the readme afterwards.

**Pull request merged**